src/components/todo_crud/TodoRemove.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <ul>
+      <li v-for="(todo, index) in todos" :key="index">
+        {{ todo }}
+      </li>
+      <Button label="Remove" @click="removeTodo(index)"></Button>
+    </ul>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import Button from '../ui/Button.vue'
+  
+const todos = ref<string[]>([]);
+const emit = defineEmits(['remove-todo']);
+const { index } = defineProps(['index']);
```

Comment:
`<script setup lang="ts">
interface Props {
  foo: string
  bar?: number
}

const props = defineProps<Props>()
</script>`


Please define props like this